iterators_task_1/student/include/impl.h

```diff
@@ -10,4 +11,18 @@
  *
  * @warning YOU MUST USE STRING STREAMS AND STREAM ITERATORS!
  */
-std::string reverseWords(const std::string& input);
+
+std::string reverseWords(const std::string& input){
+    std::stringstream ss(input);
+
+    std::string res, word;
+
+    ss >> word;
+    res = word;
+
+    while (ss >> word) {
```

Comment:
acceptable way, but try to solve with help of containers/iterators and algorithms